deepctr/layers/__init__.py

```diff
@@ -1,15 +1,15 @@
 import tensorflow as tf
 
 from .activation import Dice
-from .core import DNN, LocalActivationUnit, PredictionLayer
+from .core import DNN, LocalActivationUnit, PredictionLayer,SampledSoftmaxLayer,EmbeddingIndex,PoolingLayer
```

Comment:
these layers are in [deepmatch](https://github.com/shenweichen/DeepMatch/blob/master/deepmatch/layers/core.py), can we import them from deepmatch instead of rewriting them?